static/js/actions/course_enrollments.js

```diff
@@ -33,3 +35,16 @@ export const addCourseEnrollment = (courseId: string): Dispatcher<*> => {
       });
   };
 };
+
+export const showEnrollPayLaterSuccessMessage = (): Dispatcher<*> => {
+  return (dispatch: Dispatch) => {
+    dispatch(showEnrollPayLaterSuccess(true));
+    let promise = new Promise(function(resolve) {
+      window.setTimeout(function() {
+        dispatch(showEnrollPayLaterSuccess(false));
+        resolve('done!');
+      }, 9000);
+    });
+    return promise;
+  };
+};
```

Comment:
it is in actions actions/course_enrollments.js,

by the way done with addressing ur comments